gluecodium/src/main/resources/templates/dart/DartLambda.mustache

```diff
@@ -18,7 +18,9 @@
   ! License-Filename: LICENSE
   !
   !}}
-{{>dart/DartDocumentation}}{{>dart/DartAttributes}}
+{{>dart/DartLambdaDocs}}
+
```

Comment:
I have to leave a blank line here. Otherwise, the lambda declaration will be in the same line as the empty comment, which would hide the lambda declaration.